Src/BootCamp.Chapter1/Hangman.cs

```diff
@@ -23,18 +21,103 @@ public static class Hangman
         /// <returns>True if game was won or false if game was lost.</returns>
         public static bool Play(int lives, string wordsFile, int difficulty)
         {
+            if (difficulty < 3 || lives == 0)
```

Comment:
3 is a magic number. Please use a named constant such as ``minimumLives``

Reply:
done